docs/source/custom_task.rst

```diff
@@ -1,37 +1,139 @@
-############
-Write a Task
-############
+Tutorial: Creating a Custom Task
+================================
 
-You can create your own Flash task to if you want to solve any other deep learning problem.
+In this tutorial we will go over the process of creating a custom task,
+along with a custom data module.
 
-See this example for defining a linear classifier task:
-
-.. testcode::
+.. code:: ipython3
 
+    import flash
```

Comment:
You should introduce that you are importing the necessary dependencies. Why do we need sklearn?